rmgpy/molecule/generator.py

```diff
@@ -66,35 +66,40 @@
                  'CH4O': 'CO',
                  'CO2': 'O=C=O',
                  'CO': '[C-]#[O+]',
-                 'C2H4': 'C=C',
                  'O2': 'O=O',
                  'C': '[C]',  # for this to be in the "molecule" list it must be singlet with 2 lone pairs
-                 'SO2': 'O=S=O',
-                 'SO3': 'O=S(=O)=O',
-                 'H2SO4': 'OS(=O)(=O)O',
+                 'H2S': 'S',
                  'N2O': 'N#[N+][O-]',
                  'NH3': 'N',
                  'O3': '[O-][O+]=O',
+                 'Cl2': '[Cl][Cl]',
+                 'ClH': 'Cl',
```

Comment:
Do you think it is worthwhile to add 'HCl' in addition to, or in place of, 'ClH'?

I'm not familiar with writing hydrochloric acid as ClH

Reply:
Otherwise looks great to me.

Reply:
Following the usage of this dictionary, it is being called on [lines 282-287](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/molecule/generator.py#L284) using Molecule.getFormula(). Since the formula is in alphabetical order (except for cases where C is present, then it precedes H), the dictionary is only called with the `ClH` key. For this reason we also have `HO` instead of `OH`